tests-beta/android/MASVS-RESILIENCE/MASTG-TEST-0265.md

```diff
@@ -5,6 +5,22 @@ id: MASTG-TEST-0265
 type: [static]
 weakness: MASWE-0094
 best-practices: []
-status: draft
-note: This test checks whether the app uses StrictMode APIs, which can expose sensitive implementation details in the logs.
+status: new
 ---
+
+## Overview
+
+This test checks whether the app uses `StrictMode`, which while useful for developers to log policy violations such as disk I/O or network operations in production apps, can expose sensitive implementation details in the logs that could be exploited by attackers.
```

Comment:
```suggestion
This test checks whether the app uses `StrictMode`. While useful for developers to log policy violations such as disk I/O or network operations during development, it can expose sensitive implementation details in the logs that could be exploited by attackers.
```